microsoft-edge/devtools-guide-chromium/css/reference.md

```diff
@@ -306,114 +306,31 @@ To add a new style rule:
+By default, when adding a style rule, DevTools creates a new stylesheet named `inspector-stylesheet` in the document and then adds the new style rule in this stylesheet.
 
-*  Click and hold **New Style Rule** (![New Style Rule.](../media/new-style-rule-icon.msft.png)) to select a stylesheet to add the style rule to.
+To instead add the rule in an existing stylesheet:
 
-![Selecting a stylesheet.](../media/css-elements-styles-style-new-select-existing.msft.png)
+*  Click and hold **New Style Rule** (![New Style Rule.](../media/new-style-rule-icon.msft.png)) and then select a stylesheet from the list to add the style rule to.
 
-<!-- is the above an alternative to the below, or sequential? clarify flow/relation -->
+![Selecting a stylesheet.](../media/css-elements-styles-style-new-select-existing.msft.png)
 
 
 ### Add a style rule to a specific location
 
-To add a style rule to a specific location in the **Styles** panel:<!--is this opposed to some above approach?-->
+By default, adding a style rule by clicking on **New Style Rule** inserts the rule beneath the **element.style** rule.
 
-1. Hover on the style rule that is directly above where you want to add your new style rule.
+To add a style rule in a specific location of the **Styles** panel instead:
 
-1. [Reveal the **More Actions** toolbar](#reveal-the-more-actions-toolbar).
+1. Hover on the style rule that is directly above where you want to add your new style rule.
 
-1. Select **Insert Style Rule Below** (![Insert Style Rule Below icon.](../media/new-style-rule-icon.msft.png)).
+1. Click **Insert Style Rule Below** (![Insert Style Rule Below icon.](../media/new-style-rule-icon.msft.png)).
 
 ![Insert Style Rule Below.](../media/css-elements-styles-insert-style-rule-below.msft.png)
 
 
-<!-- ====================================================================== -->
-## Reveal the More Actions toolbar
-
-Use the **More Actions** toolbar to:
-
-*  Insert a style rule directly below the one you are focused on.
-
-*  Add a `background-color`, `color`, `box-shadow`, or `text-shadow` declaration to the style rule you are focused on.
-
-
-To reveal the **More Actions** toolbar:
-
-1. In the **Styles** panel, hover on a style rule.  **More Actions** (`...`) is revealed in the bottom-right of the style rule section.
-
-   In the following figure, hover on the `.header-holder.has-default-focus` style rule and **More Actions** is revealed in the bottom-right of the style rule section.
-
-   ![Reveal 'More Actions' (...).](../media/css-elements-styles-new-rule-styles.msft.png)
-
-1. Hover on **More Actions** (`...`) to reveal the actions mentioned above.
-
-   The **Insert Style Rule Below** action is revealed after hovering over **More Actions**.
-
-   ![The 'More Actions' toolbar.](../media/css-elements-styles-rule-more-options-insert-style-rule-below.msft.png)
-
-
-<!-- ====================================================================== -->
-## Add a background-color declaration
```

Comment:
@codepo8 - Were these UI controls removed entirely, or moved? These were items on the removed More Actions toolbar, in the **Styles** tab of the **Elements** panel in DevTools.

Add a background-color declaration - **Add Background Color**
Add a color declaration - **Add Color**
Add a box-shadow declaration - **Add Box Shadow**
Add a text-shadow declaration - **Add Text Shadow**

Reply:
Discussed offline. Yes, these were removed.